### PR TITLE
Heavy

### PR DESCRIPTION
**Pull request by AdrianoGSM**
## Type of change

- [ x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [x ] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x ] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x ] I have created the manifest.json with the required values

### CODE REVIEW

**Comment by AdrianoGSM** [comment]
Server is online, api that is returned offline https://mcstatus.io/status/java/redeheavy.com.br

File: minecraft_servers/heavy/background.png
Comment (maaaathis) [inline minecraft_servers/heavy/background.png:1]: Compress this file

File: minecraft_servers/heavy/background@2x.png
Comment (maaaathis) [inline minecraft_servers/heavy/background@2x.png:1]: Compress this file

File: minecraft_servers/heavy/manifest.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
+{
+  "server_name": "heavy",
+  "nice_name": "Rede Heavy",
+  "direct_ip": "redeheavy.com.br",
+  "server_wildcards": [
+    "%.redeheavy.com.br"
```
Comment (maaaathis) [inline minecraft_servers/heavy/manifest.json:6]: Doesn't look like a real wildcard:
image

**Review by maaaathis — CHANGES_REQUESTED** [review]

File: minecraft_servers/heavy/background.png
Comment (AdrianoGSM) [reply minecraft_servers/heavy/background.png:1]: [tinified.zip](https://github.com/user-attachments/files/16698699/tinified.zip)
I've already uploaded everything, as well as sending it here, and everything is fine about the DNS
image

IP 1: redeheavy.com.br
IP 2: jogar.redeheavy.com.br

**Review by AdrianoGSM — COMMENTED** [review]
Update images

**Pull request closed without merging**